manifests/r/RStudio/quarto/1.0.35/RStudio.quarto.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RStudio.quarto
+PackageVersion: 1.0.32
```

Comment:
```suggestion
PackageVersion: 1.0.35
```

Reply:
Yes I just saw that I missed it during the version update